code/modules/mob/living/carbon/carbon.dm

```diff
@@ -1,8 +1,10 @@
 /mob/living/carbon/Initialize()
 	//setup reagent holders
-	bloodstr = new/datum/reagents/metabolism(120, src, CHEM_INJECT)
-	touching = new/datum/reagents/metabolism(1000, src, CHEM_TOUCH)
-	reagents = bloodstr
+	if(!bloodstr && !reagents)
```

Comment:
Should split this into two `if` blocks to avoid a null reagents or bloodstr due to a partial init.

Reply:
Alright. I also modified the destroy proc to qdel bloodstr, and null out reagents. It was the other way around before and it could cause some confusion.